### PR TITLE
Apple clone

### PR DESCRIPTION
**Pull request by mariobarrioss**

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+linters:
+  stylelint:
+    config: 'stylelint.config.js'
```
Comment (bolah2009) [inline .stickler.yml:3]: ### :thinking: Have you investigate the reason why the linter is not working? The config file looks good to me.

#### I ran the [`stylelint`](https://stylelint.io/) [locally](https://github.com/microverseinc/linters-config/tree/master/css#troubleshooting) and caught no error/warning. :+1:

**Review by bolah2009 — APPROVED** [review]
# Hi @mariobarrioss  :wave: 

## Awesome Job :muscle: The [clone](https://raw.githack.com/mariobarrioss/Apple-website-clone/3ae3589cb8ad7023a08d82cd6f80752f00a8513c/index.html) looks good to me. 

#### You can merge to master. :+1: 


Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: .stickler.yml (hunk 1 above)
Comment (mariobarrioss) [reply .stickler.yml:3]: Hi Bola,

Thank you for your feedback. I'm not sure whats the issue, I'll see if it happens with other repositories in the future. I also ran stylelint locally and no issues were found.

Greetings
@mariobarrioss

**Comment by mariobarrioss** [comment]
Project approved

**Pull request closed without merging**